scripts/linters/pylint_extensions.py

```diff
@@ -1918,7 +1855,6 @@ def register(linter):
     linter.register_checker(SingleNewlineAboveArgsChecker(linter))
```

Comment:
This is also related to Docstring. This should also be moved to DocstringParameterChecker, right?

Reply:
Done.